### ISSUE TITLE
iOS change quality restarts playback

### ISSUE THREAD

**Issue by eSeal44** [sampotts/plyr#997]
### Expected behaviour
Continue playback from current time after changing quality on iOS devices

### Actual behaviour
Restarts playback from begining after changing quality on iOS devices

### Environment

- Browser: iPhone Safari
- Version: 11
- Operating System: iOS
- Version: 11.3.1

### Steps to reproduce
- on iOS devices start video playback
- change quality to other one

**Comment by sampotts** [comment]
Which version are you using? This should be resolved in the latest version but please comment if not and I will re-open.

**Comment by eSeal44** [comment]
3.3.7 from cdn.plyr.io

**Comment by sampotts** [comment]
Cool. Try 3.3.10 👍

**Comment by eSeal44** [comment]
Same result with 3.3.10 =(

**Comment by eSeal44** [comment]
Re-open please. Same trouble with 3.3.10 ... playback restarts after quality changing.

**Comment by sampotts** [comment]
Will do. Sorry about that.

**Comment by friday** [comment]
Seems like it's going to 0 while buffering, then back, with a desktop Chrome. Probably related.

**Comment by sampotts** [comment]
Yep it has to do that. When you load new media it won't know the duration until the `loadedmetadata` event is fired. A change was made to fix it but I guess it doesn't work for iOS for some reason... https://github.com/sampotts/plyr/pull/992

**Comment by friday** [comment]
https://stackoverflow.com/questions/21189958/javascript-onloadedmetadata-event-not-firing-on-ios-devices :/

**Comment by friday** [comment]
I think this is a different symptom of #857
